package.json

```diff
@@ -24,13 +23,14 @@
   },
   "homepage": "https://github.com/browserstack/webdriverio-browserstack#readme",
   "dependencies": {
-    "@wdio/cli": "^5.22.4",
-    "browserstack-local": "^1.3.7",
-    "chai": "^4.2.0",
-    "webdriverio": "^5.22.4"
+    "@wdio/cli": "^7.16.15",
+    "browserstack-local": "^1.4.8",
```

Comment:
Need this package `browserstack-local` now?

Reply:
doesnt matter, wdio uses it internally so it still be used. but removing entry from here is the ideal way